01 Styles/05 Import Materialize/README.md

```diff
@@ -0,0 +1,186 @@
+# 05 Import Materialize
+
+In this demo we will install and configure webpack to import the Google framework [Materialize](http://materializecss.com/).
+
+We will start from sample _01 Styles / 01 Custom CSS_.
+
+Summary steps:
+ - Install Materialize.
+ - Import the CSS and JS library.
+ - Use a card element from Materialize in our HTML.
+ - Check that we get errors when running webpack.
+ - Install additional loaders in order to manage fonts and other
+ files required by Materialize.
+ - Check results.
+
+# Steps to build it
+
+## Prerequisites
+
+You will need to have [Node.js](https://nodejs.org/en/) installed in your computer. If you want to follow this step guide you will need to take sample _01 Custom CSS_ as a starting point.
+
+## Steps
+
+- `npm install` to install previous sample packages:
+
+  ```
+  npm install
+  ```
+
+- Let's start by installing Materialize:
+
+  ```
+  npm install materialize-css --save
+  ```
+
+- Now, let's import the CSS and JS library in order to include it in our project:
+
+  ### ./webpack.config.js
+  ```diff
+  ...
+
+  module.exports = {
+    entry: {
+      app: './students.js',
+      appStyles: [
+        './mystyles.css',
+      ],
+      vendor: [
+        'jquery',
+  +     './node_modules/materialize-css/dist/js/materialize.js',
```

Comment:
In [package.json](https://github.com/Dogfalo/materialize/blob/master/package.json#L7), you can decide which .js is your main file which it's going to run when you add it to app or for example which webpack goes to take when you add to vendor section.